Assembly-CSharp/ModHooks.cs

```diff
@@ -2190,7 +2190,7 @@ Type type
             bool onlyEnabled = false,
             bool allowLoadError = false
         ) => ModLoader.ModInstances
-            .Where(x => onlyEnabled && !x.Enabled || !allowLoadError && x.Error != null)
+            .Where(x => !(onlyEnabled && !x.Enabled || !allowLoadError && x.Error != null))
```

Comment:
Simplify the conditional to like `(x.Enabled || !onlyEnabled) && (x.Error != null || allowLoadError)` or `(!onlyEnabled || x.Enabled) && (allowLoadError || x.Error != null)` please. The nested nots are hard to read.